### PR TITLE
Teach move_and_process.py to measure loudness for existing files.

### PR DESCRIPTION
**Pull request by petterreinholdtsen**
Process a few of the old files when there is nothing else to do,
and only process at most 6 files at the time, and for not much more
than 60 seconds.

### CODE REVIEW

File: utils/move_and_process.py [hunk 1]
```diff
@@ -254,13 +255,57 @@ def create_videofile(video_id, data):
...
+        return
+    maxtime = 60 # stop processing after this amount of seconds
+    pagesize = 2 # Process this amount of video file per format at the time
+    start = time.time()
+    for fsname in ('original', 'broadcast', 'theora'):
+        response = rq('GET', '/videofiles/', params={'format__fsname': fsname,
+                                                     'integrated_lufs__isnull': True,
+                                                     'page_size': pagesize,
+                                                     'order': '-video'})
+        for data in response.json()['results']:
```
[the remark this answers is not in the document]
Comment (petterreinholdtsen) [reply utils/move_and_process.py]: Good idea.

File: utils/move_and_process.py [hunk 2]
```diff
@@ -254,13 +255,57 @@ def create_videofile(video_id, data):
...
+    if args.no_api:
+        logging.debug("NO API - creating videofile %s -> %r" % (video_id, data))
+        return
+    rq('PATCH', '/videofiles/', data=data)
+
+def measure_loudness(watch_dir, move_to_dir):
+    """Measure loudness for all video files missing it, a few at the time.
+
+Process latest videos first.
+"""
```
[the remark this answers is not in the document]
Comment (petterreinholdtsen) [reply utils/move_and_process.py]: Done.

File: utils/move_and_process.py [hunk 3]
```diff
@@ -254,13 +255,57 @@ def create_videofile(video_id, data):
     data.update({'video': video_id})
     rq('POST', '/videofiles/', data=data)
 
+def update_videofile(data):
+    if args.no_api:
+        logging.debug("NO API - creating videofile %s -> %r" % (video_id, data))
+        return
+    rq('PATCH', '/videofiles/', data=data)
```
[the remark this answers is not in the document]
Comment (petterreinholdtsen) [reply utils/move_and_process.py]: Fixed.

File: utils/move_and_process.py [hunk 4]
```diff
@@ -256,10 +256,11 @@ def create_videofile(video_id, data):
     rq('POST', '/videofiles/', data=data)
 
 def update_videofile(data):
+    videofile_id = data['id']
     if args.no_api:
-        logging.debug("NO API - creating videofile %s -> %r" % (video_id, data))
+        logging.debug("NO API - updating videofile %s -> %r" % (videofile_id, data))
         return
-    rq('PATCH', '/videofiles/', data=data)
+    rq('PATCH', '/videofiles/%s' % data['id'], data=data)
```
Comment (odinho) [inline utils/move_and_process.py]: I'd reuse the `videofile_id` you created above.

File: utils/move_and_process.py [hunk 5]
```diff
@@ -241,11 +241,11 @@ def _update_video(video_id, data):
         return
     response = rq('PATCH', '/videos/%s' % video_id, data=data)
 
-def get_videofiles(video_id):
+def get_videofiles(params):
     if args.no_api:
         logging.debug("NO API - get videofiles %s -> []" % video_id)
```
[the remark this answers is not in the document]
Comment (petterreinholdtsen) [reply utils/move_and_process.py]: Right.  I missed the video_id use in the log line.  Fixed.

File: utils/move_and_process.py (hunk 4 above)
Comment (petterreinholdtsen) [reply utils/move_and_process.py]: Good idea.

**Pull request merged**